src/h5web/vis-packs/core/heatmap/HeatmapAxisSystem.tsx

```diff
@@ -0,0 +1,60 @@
+import type { ReactNode } from 'react';
+import { useThree } from '@react-three/fiber';
+import AxisSystem from '../shared/AxisSystem';
+import type { AxisOffsets, AxisConfig } from '../models';
+import { computeSameRatioDomains } from '../utils';
+
+interface Props {
+  axisOffsets: AxisOffsets;
+  abscissaConfig: AxisConfig;
+  ordinateConfig: AxisConfig;
+  imageRatio?: number;
+  title?: string;
+  children: ReactNode;
+}
+
+function HeatmapAxisSystem(props: Props) {
+  const {
+    axisOffsets,
+    abscissaConfig,
+    ordinateConfig,
+    imageRatio,
+    title,
+    children,
+  } = props;
```

Comment:
To save a bit of space, you could spread the props that get forwarded in `...forwardedProps` (`children` `axisOffsets` and `title`), and then return:

```tsx
<AxisSystem
  abscissaConfig={abscissaConfig}
  ordinateConfig={ordinateConfig}
  {...forwardedProps}
/>
```